OAK-Marketplace-Emotion-Recognition/utils/inference.py

```diff
@@ -0,0 +1,26 @@
+from emotion_recognition_retail import process_frame
+import cv2
+import numpy as np
+from visualization import statistics_update
+
+
+def process_cam(model, emotion_analyzer, show: bool = True):
+    visualization_results = []
+    model.add_cam_to_pipeline()
```

Comment:
Please, add comments because it's a snippet and it should be understandable for readers. You could look at snippets in this post